### DISCUSSION TITLE
Apple Music

### DISCUSSION THREAD

**Discussion by Iwo24pl** [catppuccin/catppuccin#2370] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Port themes from spotify to apple music on windows

### Any additional comments?

-

### Submission Guidelines

- [x] I have read and followed Catppuccin's submission guidelines :D

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!
